recipe/meta.yaml

```diff
@@ -1,37 +1,49 @@
-{%set name = "apache-libcloud" %}
-{%set version = "1.3.0" %}
-{%set hash_type = "sha256" %}
-{%set hash_val = "198d86a69f889ded7c023567280f6c479862da5a3d82f70eba7c208bba07b395" %}
+{% set name = "apache-libcloud" %}
+{% set version = "3.8.0" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash_val }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache-libcloud-{{ version }}.tar.gz
+  sha256: 75bf4c0b123bc225e24ca95fca1c35be30b19e6bb85feea781404d43c4276c91
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - python
     - setuptools
```

Comment:
upstream has specified pinnings for setuptools and wheel - do you know why they pinned in the first place?

Reply:
The maintainers tried to test python 3.12 https://github.com/apache/libcloud/commit/ff8059a84ad70eb67ac9c473a0762b9a9efe3ae7. I think they wanted to migrate from setup.py to pyproject.toml and this migration was careful enough. I would ignore their pinnings for setuptools & wheel

Reply:
Sometimes that's a way to to keep their CI stable.